__tests__/app.test.js

```diff
@@ -0,0 +1,17 @@
+const app = require('../app')
+const supertest = require('supertest')
+
+describe('Topics endpoints', ()=>{
+    test('GET /api/topics should return a json containing all topics with status 200', async ()=>{
+        let apiRes = [ { slug: 'coding', description: 'Code is love, code is life' },
+        { slug: 'football', description: 'FOOTIE!' },
+        { slug: 'cooking',
+          description: 'Hey good looking, what you got cooking?' }]
+        
+        await supertest(app).get('/api/topics')
```

Comment:
like the use of async/await here Lucca - you could also refactor the use of `.then` if you want to take this further